modules/payment/src/Entity/Payment.php

```diff
@@ -24,6 +24,7 @@
  *   bundle_plugin_type = "commerce_payment_type",
  *   handlers = {
  *     "access" = "Drupal\commerce_payment\PaymentAccessControlHandler",
+ *     "event" = "Drupal\commerce_payment\Event\PaymentEvent",
```

Comment:
I don't think this is necessary if we state_machine

Reply:
It is, because payments do not use transitions.